web/src/i18n/config.ts

```diff
@@ -57,7 +57,8 @@ i18next
           lineage: {
             empty_title: 'No node selected',
             empty_body: 'Try selecting a node through search or the jobs or datasets page.',
-            graph_depth_title: 'Graph Depth'
+            graph_depth_title: 'Graph Depth',
+            full_graph_label: 'Full Graph'
```

Comment:
```
full_graph_label: 'Show Critical Path'
```

Reply:
Eh, this just starts to feel like a lot of words to express something relatively simple. I tend to err on the side of "less is more" with these things. I really like the [ArgoCD](https://argo-cd.readthedocs.io/) version of this where the text labels show up as a tooltip when you hover over this options. I think that helps preserve the real-estate the configs take up. Maybe we want to think about something like that if we plan to add more configs to this part of the screen.

<img width="774" alt="image" src="https://github.com/MarquezProject/marquez/assets/36255627/0d9cb48f-a96d-43c4-94fc-4bb3b6edd689">

Reply:
Yeah I’m in two minds about that. Like the idea of having a pop up on hover to display basic ( maybe even facets like data quality?) info about the node. Maybe we need an additional menu on the header for global type variables like display and depth?

Reply:
As a follow up, the lineage API should support the different graph options (`Full`, `Critical`) to avoid filtering on the UI. But, in the UI is a great first step!